benchmarks/backend/heatmap/setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Start the database and run migrations
+cd backend
+docker run -d --name postgis -e POSTGRES_PASSWORD=permaplant -e POSTGRES_USER=permaplant -p 5432:5432 postgis/postgis:13-3.1 -c log_min_duration_statement=0 -c log_statement=all
+sleep 10
+LC_ALL=C diesel setup
+LC_ALL=C diesel migration run
+
+# Insert data
+cd ../scraper && npm run insert
+printf "\nNow run the backend in a second shell.\n"
+read -n 1 -p "Press any key to continue!"
+../benchmarks/backend/heatmap/insert_data-large_map.sh $1 $2
+printf "\n\nStop the backend.\n"
+read -n 1 -p "Press any key to continue!"
+
+# Start the backend
+cd ../backend
```

Comment:
I don't like modifying files like this in scripts. If I were to append it to the Cargo.toml I would need to remove it again at the end of the script. However if I interrupt the script it might get messy, so I'd rather the user inserts it manually.

Reply:
Yeah, without a proper TOML tool it is not a good idea. Elektra can do such things but I agree this is over-the-top for this :wink: